language/de.json

```diff
@@ -0,0 +1,372 @@
+	"Negative ADM Guidance Scaler": "Skalierer für negative ADM-Führung",
+	"The scaler multiplied to negative ADM (use 1.0 to disable). ": "Der Skalierer multipliziert mit negativem ADM (1.0 verwenden, um zu deaktivieren).",
+	"ADM Guidance End At Step": "ADM-Führungsende beim Schritt",
+	"When to end the guidance from positive/negative ADM. ": "Wann die Führung von positivem/negativem ADM enden soll.",
+	"Refiner swap method": "Veredler-Austauschmethode",
+	"joint": "gemeinsam",
+	"separate": "separat",
+	"vae": "vae",
+	"CFG Mimicking from TSNR": "CFG-Nachahmung von TSNR",
+	"Enabling Fooocus's implementation of CFG mimicking for TSNR (effective when real CFG > mimicked CFG).": "Aktivierung der Fooocus-Implementierung von CFG-Nachahmung für TSNR (wirksam, wenn echtes CFG > nachgeahmtes CFG).",
+	"Sampler": "Sampler",
+	"dpmpp_2m_sde_gpu": "dpmpp_2m_sde_gpu",
+	"Only effective in non-inpaint mode.": "Nur wirksam im Nicht-Inpaint-Modus.",
+	"euler": "euler",
+	"euler_ancestral": "euler_ancestral",
+	"heun": "heun",
+	"dpm_2": "dpm_2",
+	"dpm_2_ancestral": "dpm_2_ancestral",
+	"lms": "lms",
+	"dpm_fast": "dpm_fast",
+	"dpm_adaptive": "dpm_adaptive",
+	"dpmpp_2s_ancestral": "dpmpp_2s_ancestral",
+	"dpmpp_sde": "dpmpp_sde",
+	"dpmpp_sde_gpu": "dpmpp_sde_gpu",
+	"dpmpp_2m": "dpmpp_2m",
+	"dpmpp_2m_sde": "dpmpp_2m_sde",
+	"dpmpp_3m_sde": "dpmpp_3m_sde",
+	"dpmpp_3m_sde_gpu": "dpmpp_3m_sde_gpu",
+	"ddpm": "ddpm",
+	"ddim": "ddim",
+	"uni_pc": "uni_pc",
+	"uni_pc_bh2": "uni_pc_bh2",
+	"Scheduler": "Zeitplaner",
+	"karras": "karras",
+	"Scheduler of Sampler.": "Zeitplaner des Samplers.",
+	"normal": "normal",
+	"exponential": "exponentiell",
+	"sgm_uniform": "sgm_uniform",
+	"simple": "einfach",
+	"ddim_uniform": "ddim_uniform",
+	"Forced Overwrite of Sampling Step": "Erzwungenes Überschreiben des Sampling-Schritts",
+	"Set as -1 to disable. For developer debugging.": "Auf -1 setzen, um zu deaktivieren. Für Entwickler-Debugging.",
+	"Forced Overwrite of Refiner Switch Step": "Erzwungenes Überschreiben des Refiner-Wechselschritts",
+	"Forced Overwrite of Generating Width": "Erzwungenes Überschreiben der Erzeugungsbreite",
+	"Set as -1 to disable. For developer debugging. Results will be worse for non-standard numbers that SDXL is not trained on.": "Auf -1 setzen, um zu deaktivieren. Für Entwickler-Debugging. Die Ergebnisse werden für nicht standardmäßige Zahlen, auf die SDXL nicht trainiert ist, schlechter sein.",
+	"Forced Overwrite of Generating Height": "Erzwungenes Überschreiben der Erzeugungshöhe",
+	"Forced Overwrite of Denoising Strength of \"Vary\"": "Erzwungenes Überschreiben der Entstörungsstärke von \"Vary\"",
+	"Set as negative number to disable. For developer debugging.": "Als negative Zahl einstellen, um zu deaktivieren. Für Entwickler-Debugging.",
+	"Forced Overwrite of Denoising Strength of \"Upscale\"": "Erzwungenes Überschreiben der Entstörungsstärke von \"Upscale\"",
```

Comment:
Erzwungenes Überschreiben der Entrauschungsstärke von \"Upscale\"

Reply:
This suggestion has been implemented.